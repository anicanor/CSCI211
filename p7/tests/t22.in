echo t22: Insert days of week into tree, then call balanced, rebalance, balanced
insert Friday
insert Monday
insert Saturday
insert Sunday
insert Wednesday
insert Thursday
insert Tuesday
balanced
echo calling rebalance
rebalance
balanced
