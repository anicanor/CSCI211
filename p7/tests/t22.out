t22: Insert days of week into tree, then call balanced, rebalance, balanced
Tree is not balanced.
calling rebalance
Tree is balanced.
